docs/api-reference/globe-view.md

```diff
@@ -37,7 +39,7 @@ To render, `GlobeView` needs to be used together with a `viewState` with the fol
 - `zoom` (`Number`) - zoom level
 - `maxZoom` (`Number`, optional) - max zoom level. Default `20`.
 - `minZoom` (`Number`, optional) - min zoom level. Default `0`.
-- `resolution` (`Number`, optional) - the resolution at which to turn flat features into 3D meshes, in degrees. Default `10`.
+- `resolution` (`Number`, optional) - the resolution at which to turn flat features into 3D meshes, in degrees. Smaller numbers will generate more detailed mesh. Default `10`.
```

Comment:
It is strange to me that this is on the view state rather than a global state. Same layer can be shown by different view states. Having special tesselation means that if I want a split view, one globe and one flat view, there are going to be issues?

Reply:
Grid-split features can be rendered in a non-globe view, but not the other way around.

If you have multiple views, then there might be a racing condition depending on which viewport got used to initialize the layer. I will add this as a known issue to the documentation.